addon/components/mdc-textfield.js

```diff
@@ -212,12 +230,7 @@ export default Ember.Component.extend(MDCComponent, {
         get(component, 'inputKeydownHandlers').removeObject(handler);
       },
       getNativeInput() {
-        const  value = get(component, 'value');
-        return {
-          value,
-          disabled: get(component, 'disabled'),
-          checkValidity: () => get(component, 'valid')
-        };
+        return getElementProperty(component, 'querySelector', () => [null])('input, textarea')[0];
```

Comment:
Why do you need to pass in `() => [null]`. Just curious.

Reply:
Good catch - [`Element.querySelector()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/querySelector) returns a single element, not a NodeList.